tests/Liquid/FilterbankTest.php

```diff
@@ -166,6 +170,30 @@ public function testObjectFilter()
 
 		$var = new Variable('var | static_test');
 		$this->assertEquals('worked', $var->render($this->context));
+
+		$var = new Variable('var | __construct');
+		$this->assertEquals('1000', $var->render($this->context));
+	}
+
+	public function testObjectFilterDontCallConstruct()
+	{
+		$this->context->set('var', 1000);
+		$this->context->addFilters(new \ClassFilter());
+
+		$filterbankReflectionClass = new \ReflectionClass(Context::class);
+		$methodMapProperty = $filterbankReflectionClass->getProperty('filterbank');
+		$methodMapProperty->setAccessible(true);
+		$filterbank = $methodMapProperty->getValue($this->context);
+
+		$filterbankReflectionClass = new \ReflectionClass(Filterbank::class);
+		$methodMapProperty = $filterbankReflectionClass->getProperty('methodMap');
+		$methodMapProperty->setAccessible(true);
+		$methodMap = $methodMapProperty->getValue($filterbank);
+
+		$this->assertArrayNotHasKey('__construct', $methodMap);
```

Comment:
@sanmai I think throw exception to test code is not best practice. That's why I test / check with ReflectionClass that __construct was not registered in the Filterbank as a possible filter. What do you think?

Reply:
Yeah, that's way more explicit and better!